geometry_msgs/PoseStamped   camera_pose
---
float32[] poses